README.md

```diff
@@ -1 +1,3 @@
 # hw01
+
+Hello World! - Patrick Taylor
```

Comment:
You should introduce a bit more about you and try to use 4 elements of markdown syntax. Anyway, a good start!